packages/auth/amplify_auth_cognito/pubspec.yaml

```diff
@@ -20,7 +20,7 @@ platforms:
 
 dependencies:
   amplify_auth_cognito_android: 1.0.0-next.0+1
-  amplify_auth_cognito_dart: 0.1.2
+  amplify_auth_cognito_dart: ^0.1.3
```

Comment:
question: why not 0.1.4?

Reply:
Could have been either. 2 small releases bundled here for auth_dart, but they are both compatible with each other.